src/routes/stories/[id]/+page.svelte

```diff
@@ -10,87 +10,137 @@
 			duration: 1.5
 		});
 
-			gsap.from('.story__content p', {
-				delay: .5,
-				x: -800,
-				ease: "back.out(1)",
-				duration: 3,
-			});
+		gsap.from('.story__content *:nth-child(1)', {
+			delay: .5,
+			x: -900,
+			ease: "back.out(.6)",
+			duration: 3,
+		});
+
+		gsap.from('.story__content *:nth-child(2)', {
+			delay: .75,
+			x: 900,
+			ease: "back.out(.6)",
+			duration: 3,
+		});
+
+		gsap.from('.story__content *:nth-child(3)', {
+			scrollTrigger: {
+				trigger: '.story__content *:nth-child(3)',
+				toggleActions:"restart none restart restart"
+			},
+			delay: .2,
+			x: -100,
+			ease: "back.out(1)",
+			duration: 1.25,
+		});
+
+		gsap.from('.story__content *:nth-child(4)', {
+			scrollTrigger: {
+				trigger: '.story__content *:nth-child(4)',
+				toggleActions:"restart none restart restart"
+			},
+			delay: .2,
+			x: 100,
+			ease: "back.out(1)",
+			duration: 1.25,
+		});
+
+		gsap.from('.story__content *:nth-child(5)', {
+			scrollTrigger: {
+				trigger: '.story__content *:nth-child(5)',
+				toggleActions:"restart none restart restart"
+			},
+			delay: .2,
+			x: -100,
+			ease: "back.out(1)",
+			duration: 1.25,
+		});
+
+		gsap.from('.story__content *:nth-child(6)', {
+			scrollTrigger: {
+				trigger: '.story__content *:nth-child(6)',
+				toggleActions:"restart none restart restart"
+			},
+			delay: .2,
+			x: 100,
+			ease: "back.out(1)",
+			duration: 1.25,
+		});
+
+		gsap.from('.story__content *:nth-child(7)', {
+			scrollTrigger: {
+				trigger: '.story__content *:nth-child(7)',
+				toggleActions:"restart none restart restart"
+			},
+			delay: .2,
+			x: -100,
+			ease: "back.out(1)",
+			duration: 1.25,
+		});
+
+		gsap.from('.story__content *:nth-child(8)', {
+			scrollTrigger: {
+				trigger: '.story__content *:nth-child(8)',
+				toggleActions:"restart none restart restart"
+			},
+			delay: .2,
+			x: 100,
+			ease: "back.out(1)",
+			duration: 1.25,
+		});
+
+		gsap.from('.story__content *:nth-child(9)', {
+			scrollTrigger: {
+				trigger: '.story__content *:nth-child(9)',
+				toggleActions:"restart none restart restart"
+			},
+			delay: .2,
+			x: -100,
+			ease: "back.out(1)",
+			duration: 1.25,
+		});
+
+		gsap.from('.story__content *:nth-child(10)', {
+			scrollTrigger: {
+				trigger: '.story__content *:nth-child(10)',
+				toggleActions:"restart none restart restart"
+			},
+			delay: .2,
+			x: 100,
+			ease: "back.out(1)",
+			duration: 1.25,
+		});
+
+		gsap.from('.story__content *:nth-child(11)', {
+			scrollTrigger: {
+				trigger: '.story__content *:nth-child(11)',
+				toggleActions:"restart none restart restart"
+			},
+			delay: .2,
+			x: -100,
+			ease: "back.out(1)",
+			duration: 1.25,
+		});
+
+		gsap.from('.story__content *:nth-child(12)', {
+			scrollTrigger: {
+				trigger: '.story__content *:nth-child(12)',
+				toggleActions:"restart none restart restart"
+			},
+			delay: .2,
+			x: 100,
+			ease: "back.out(1)",
+			duration: 1.25,
+		});
 	});
-	// onMount(() => {
```

Comment:
Remove unused code

Reply:
Er is geen unused code voor de onMount. Ik zie geen problemen op regel 27.

Reply:
Oh ja my bad, merge maar